practice-project-ui/src/Calculator.tsx

```diff
@@ -1,25 +1,63 @@
 import * as React from "react";
+// import axios from 'axios';
```

Comment:
I wouldn't commit commented out code unless you have a good reason. I understand that you've done the work for the next Jira ticket but you shouldn't do this

Reply:
Yea for sure. I'm just trying to simulate not having done it yet without adding extra work for myself. Are you happy for me to leave it in?